src/task.c

```diff
@@ -512,12 +532,57 @@ void jl_init_tasks(void) JL_GC_DISABLED
                                 jl_any_type,
                                 jl_any_type),
                         0, 1, 7);
+#else /* JULIA_ENABLE_PARTR */
+    jl_task_type = (jl_datatype_t*)
+        jl_new_datatype(jl_symbol("Task"), NULL, jl_any_type, jl_emptysvec,
+                        jl_perm_symsvec(14,
+                                        "storage",
+                                        "state",
+                                        "result",
+                                        "exception",
+                                        "backtrace",
+                                        "logstate",
+                                        "code",
+                                        "redentry",
+                                        "cq_head",
+                                        "cq_lock_owner",
+                                        "cq_lock_count",
+                                        "next",
+                                        "parent",
+                                        "redresult"),
+                        jl_svec(14,
+                                jl_any_type,
+                                jl_sym_type,
+                                jl_any_type,
+                                jl_any_type,
+                                jl_any_type,
+                                jl_any_type,
+                                jl_any_type,
+                                jl_any_type,
+                                jl_any_type,
+                                jl_int64_type,
```

Comment:
This `jl_int64_type` looks fishy. It's a part of `jl_mutex_t` which contains `volatile unsigned long owner` which is 32 bits on linux32.